server/etc/pulp/server.conf

```diff
@@ -164,7 +163,6 @@
 
 [data_reaping]
 # reaper_interval: 0.25
-# archived_calls: 0.5
```

Comment:
On 06/09/2015 04:24 PM, bmbouter wrote:
> You're saying the other ones marked as deprecated are still used while
> this one is not.

I don't know if the above is true or not. I was just saying that for
this particular setting, there's not a practical reason to mark it
deprecated since it's actually not used (and thus, removed).

-- 
Randy Barlow